dockerfiles/docker-compose.yml

```diff
@@ -37,7 +37,7 @@ services:
   wrangler:
     build:
       context: ${PWD}
-      dockerfile: ${PWD}/dockerfiles/Dockerfile.wrangler
+      dockerfile: ${PWD}/common/dockerfiles/Dockerfile.wrangler
     volumes:
       - ${PWD}/dockerfiles/force-readthedocs-addons.js:/usr/src/app/docker/force-readthedocs-addons.js
```

Comment:
Don't we still need this file under .com?

Reply:
🤔 Hrm, good point. I think this file will be shared in `common/` then since it will be the same.